targets/TARGET_WICED/TARGET_WISE1530_F412RE/WISE1530Interface.cpp

```diff
@@ -0,0 +1,340 @@
+/* WISE1530 implementation of NetworkInterfaceAPI
+ * Copyright (c) 2015 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "WISE1530Interface.h"
+
+#include "mbed.h"
```

Comment:
dont include this header file outside of user space

Reply:
Oh, is this a problem in cpp files? There's no risk of namespace polution since it's self contained in the cpp file. If anything this would help protect against filename changes.

Reply:
As I understand it, a `name.cpp` file should only ever include its `name.h` file, for stylistic reasons, not technical.